packages/generator/src/operations/parameters-interface.spec.ts

```diff
@@ -7,6 +7,10 @@ import {
 import { parametersInterface } from './parameters-interface';
 
 describe('parameters-interface', () => {
+  it('function parameter interface for bound function', () => {
```

Comment:
[req] left over?

Reply:
looks like, @FrankEssenberger can you comment on that?

Reply:
ah, look at my comment below. I wanted to add a test that the parameter object contains the expected values but forgot about it. Feel free to add a test.

Reply:
I've tried adding a test, but the function seems to do nothing different for bound/unbound.. I don't see additional value in that test.